scripts/stabilization.py

```diff
@@ -170,7 +177,7 @@ class StabilizationParameter : public Expression
                 // iterate over spatial dimensions
                 for(unsigned int d=0; d<dim; ++d) {
                     // compute (beta_d * dJ_f / x_d)| at x_k (k-th node on element)
-                    derivatives[i][f] += 0.5 * vals_beta_all[i][d] * basis_derivatives_matrix[i][k * dim + d] * pow(vals_J_all[k][f], 2);
+                    derivatives[i][f] += vals_beta_all[i][d] * basis_derivatives_matrix[i][k * dim + d] * vals_J_all[k][f];
```

Comment:
If `vals_beta_all` refers to the transport velocity, the designation `derivatives` is in my opinion misleading.